itests/eth_balance_test.go

```diff
@@ -0,0 +1,71 @@
+package itests
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/chain/types/ethtypes"
+	"github.com/filecoin-project/lotus/itests/kit"
+)
+
+func TestEthGetBalanceExistingF4address(t *testing.T) {
+	blockTime := 100 * time.Millisecond
+	client, _, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC())
+	ens.InterconnectAll().BeginMining(blockTime)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	_, ethAddr, deployer := client.EVM().NewAccount()
+
+	fundAmount := types.FromFil(10)
```

Comment:
I believe it is valid to send 0 FIL to an account, which should create it (with 0 balance). That might be an interesting quick case to add to this test.

Reply:
done